code/__HELPERS/_logging.dm

```diff
@@ -113,8 +113,15 @@
 		WRITE_LOG(GLOB.world_attack_log, "ATTACK: [text]")
 
 /proc/log_manifest(ckey, datum/mind/mind,mob/body, latejoin = FALSE)
-	if (CONFIG_GET(flag/log_manifest))
-		WRITE_LOG(GLOB.world_manifest_log, "[ckey] \\ [body.real_name] \\ [mind.assigned_role] \\ [mind.special_role ? mind.special_role : "NONE"] \\ [latejoin ? "LATEJOIN":"ROUNDSTART"]")
+    if (CONFIG_GET(flag/log_manifest))
+        if(istype(body, /mob/living/carbon))
+            var/mob/living/carbon/M = body
+            if(M?.dna?.species)
+                WRITE_LOG(GLOB.world_manifest_log, "[ckey] \\ [body.real_name] \\ [mind.assigned_role] \\ [mind.special_role ? mind.special_role : "NONE"] \\ [latejoin ? "LATEJOIN":"ROUNDSTART"] \\ [format_text(M?.dna?.species?.name)]")
+            else
+                WRITE_LOG(GLOB.world_manifest_log, "[ckey] \\ [body.real_name] \\ [mind.assigned_role] \\ [mind.special_role ? mind.special_role : "NONE"] \\ [latejoin ? "LATEJOIN":"ROUNDSTART"] \\ Something broke, yell on ghub")
```

Comment:
dna and species can acceptably be null. most specifically dna, before a mob is fully initialized. I would probably just exclude the "this is a bug" line.

Reply:
Thing is, manifest logs run after the mob was initialized. But sure, I'll remove it.